models/comments.js

```diff
@@ -1,9 +1,25 @@
 const db = require("../db/connection.js");
 
-exports.selectCommentsArticleById = (articleId) => {
+exports.selectCommentsByArticleId = (articleId) => {
   return db
     .query(`SELECT * FROM comments WHERE article_id = $1;`, [articleId])
     .then(({ rows }) => {
       return rows;
     });
 };
+
+exports.insertComment = (articleId, { username: author, body }) => {
+  return db
+    .query(
+      `
+    INSERT INTO comments 
+    (article_id, author, body, votes, created_at)
+    VALUES
+    ($1, $2, $3, 0, NOW())
```

Comment:
Take a look at the table creations in the seed - you may find votes and created_at are already handled for new insertions